tests/test_job.py

```diff
@@ -49,6 +51,20 @@ def test_delete_job(nomad_setup):
     test_register_job(nomad_setup)
 
 
+@pytest.mark.skipif(os.environ.get("NOMAD_VERSION") in 
```

Comment:
I like this change, I didn't feel good about a static list that someone would have to remember to update.